api/Navigator.json

```diff
@@ -2285,16 +2286,18 @@
               }
             ],
             "firefox_android": [
+              {
+                "version_added": "79",
+                "partial_implementation": true,
+                "notes": "Vibration is disabled. If the window is visible, then <code>navigator.vibrate()</code> returns <code>true</code>, but no vibration takes place (regardless of hardware support). See <a href='https://bugzil.la/1591113'>bug 1591113</code>."
+              },
               {
                 "version_added": "16",
+                "version_removed": "79",
                 "notes": [
                   "Until Firefox 26 included, when the vibration pattern was too long or any of its elements too large, Firefox threw an exception instead of returning <code>false</code> (<a href='https://bugzil.la/884935'>bug 884935</a>).",
                   "From Firefox 32 onwards, when the vibration pattern is too long or any of its elements too large, it returns <code>true</code> but truncates the pattern (<a href='https://bugzil.la/1014581'>bug 1014581</a>)."
                 ]
-              },
```

Comment:
suggestion: instead of removing the `moz` prefix support object here, I propose we add `"version_removed": true`" instead.

`mozVibrate` definitely existed historically, but I don't know when it was removed. We could test to find out, but I think it's out of scope to for this PR. I'd like to see the really big thing (vibration not doing anything on Android) get merged before investigating a historical footnote. 🙂

Reply:
This is unresolved still. We need something like this in the final array:

```
              {
                "version_added": "11",
                "version_removed": true,
                "prefix": "moz"
              }
```

Reply:
easy enough